### DISCUSSION TITLE
Timed automation renewal certificate

### DISCUSSION THREAD

**Discussion by chenqimiao** [NginxProxyManager/nginx-proxy-manager#4250] [Q&A] [answered]
I am a beginner and I have a question to ask the developers in the community. How can I configure scheduled automated renewal of SSL certificates? For example, in the first ten days before the expiration date, scheduled tasks automatically renew for me.
Looking forward to everyone's reply. Thank you

**Comment by felixschndr** [comment, marked as answer]
I guess you don't need to do that. At least for me it auto-regenerates the certs. I use the LetsEncrypt DNS challenge on an IONOS domain and NPM takes care of renewing the cert some time before it expires. Never had any issues with that

**Reply by chenqimiao** [reply]
Thanks for your answer !
